src/runtime-rs/crates/resource/src/network/network_with_netns.rs

```diff
@@ -82,11 +82,11 @@ impl NetworkWithNetns {
 
 #[async_trait]
 impl Network for NetworkWithNetns {
-    async fn setup(&self, h: &dyn Hypervisor) -> Result<()> {
+    async fn setup(&self, _h: &dyn Hypervisor) -> Result<()> {
```

Comment:
if this argument is not needed, I think you can just remove it.

Reply:
good idea!